docs/source/initialsetup/mameintro.rst

```diff
@@ -1,43 +1,71 @@
 An Introduction to MAME
 =======================
 
-MAME, formerly was an acronym which stood for Multi Arcade Machine Emulator, documents and reproduces through emulation the inner components of arcade machines, computers, consoles, chess computers, calculators, and many other types of electronic amusement machines. As a nice side-effect, MAME allows to use on a modern PC those programs and games which were originally developed for the emulated machines.
+MAME, formerly was an acronym which stood for Multi Arcade Machine Emulator,
+documents and reproduces through emulation the inner components of arcade
+machines, computers, consoles, chess computers, calculators, and many other
+types of electronic amusement machines. As a nice side-effect, MAME allows to
+use on a modern PC those programs and games which were originally developed for
+the emulated machines.
 
-At one point there were actually two separate projects, MAME and MESS. MAME covered arcade machines, while MESS covered everything else. They are now merged into the one MAME.
+At one point there were actually two separate projects, MAME and MESS. MAME
+covered arcade machines, while MESS covered everything else. They are now
+merged into the one MAME.
 
-MAME is mostly programmed in C with some core components in C++. MAME can currently emulate over 32000 individual systems from the last 5 decades.
+MAME is mostly programmed in C with some core components in C++. MAME can
+currently emulate over 32000 individual systems from the last 5 decades.
 
 
 Purpose of MAME
 ===============
 
-The primary purpose of MAME is to preserve decades of arcade, computer, and console history. As technology continues to rush forward, MAME prevents these important “vintage” systems from being lost and forgotten.
+The primary purpose of MAME is to preserve decades of arcade, computer, and
+console history. As technology continues to rush forward, MAME prevents these
+important “vintage” systems from being lost and forgotten.
 
 
 Systems Emulated by MAME
 ========================
 
 
-ProjectMESS contains a complete list of the systems currently emulated. As you will notice, being supported does not always mean that the status of the emulation is perfect. You may want
+ProjectMESS contains a complete list of the systems currently emulated. As you
```

Comment:
I have no idea. I need to actually look into this.

Reply:
It still exists, but it hasn't been updated in many years. @etabeta78 was project lead of it.
http://www.progettoemma.net/mess/

It would be better to use Arcade Database (ADB) at http://adb.arcadeitalia.net/ , I think, though I am not certain if it has all the features of ProjectMESS, I think it might.

Reply:
This will be fixed shortly, I believe.